spyder/plugins/help/utils/static/dark_css/default.css

```diff
@@ -425,3 +425,53 @@ div.admonition p.admonition-title {
 .panel-danger > .panel-body {    
     background-color: #E74C3C;
 }
+
+/* --- Scrollbar -- */
+
+::-webkit-scrollbar {
+    background-color: #19232D;
+    width: 14px;
+    margin: 16px 2px 16px 2px;
+    border: 1px solid #32414B;
+    border-radius: 4px;
+}
+
+::-webkit-scrollbar-thumb {
+    background-color: #787878;
+    border: 1px solid #32414B;
+    min-height: 8px;
+    border-radius: 10px;
+    width: 1px;
+}
+
+::-webkit-scrollbar-thumb:hover {
+    background-color: #148CD2;
+    border: 1px solid #148CD2;
+    border-radius: 10px;
+    min-height: 8px;
+    width: 1px;
+}
+
+::-webkit-scrollbar-button:increment {
+    background-repeat: no-repeat;
+    background-image: url(../images/down_arrow_disabled.png);
```

Comment:
@dalthviz, where did you get these images from?

Please remember that if you didn't create them, you need to be sure they're compatible with the MIT license and add an entry in `NOTICE.txt`.

Reply:
Yep, I have to add `QDarkStyle` in the `NOTICE.txt` I took them from there